javascript/stimulus_reflex.js

```diff
@@ -22,6 +21,17 @@ let actionCableConsumer
 
 const promises = {}
 
+function uuidv4 () {
```

Comment:
I'm actually changing all function definitions to use function expressions so they're block scoped and a little more strict. I'd prefer to see this defined like so.

```js
const uuidv4 = () => {
```

Reply:
Thank you, now I learnt something new.

> Function expressions are invoked to avoid polluting the global scope. Instead of your program being aware of many different functions, when you keep them anonymous, they are used and forgotten immediately.